vllm/attention/backends/torch_sdpa.py

```diff
@@ -221,8 +221,8 @@ def _make_alibi_bias(
         bias = bias[None, :] - bias[:, None]
 
         num_heads = alibi_slopes.shape[0]
-        bias = bias[None, :].expand(num_heads, prompt_len, prompt_len)
-        bias.mul_(alibi_slopes[:, None, None])
+        bias = bias[None, :].expand(num_heads, prompt_len, prompt_len)\
+                .mul(alibi_slopes[:, None, None])
```

Comment:
Why do we change this?

Reply:
Because ```expand``` outputs a tensor view doesn't support inplace operation.